pkg/test/framework/components/ingress/ingress.go

```diff
@@ -30,12 +39,21 @@ type Instance interface {
 	Address() string
 
 	//  Call makes an HTTP call through ingress, where the URL has the given path.
-	Call(path string) (CallResponse, error)
-	CallOrFail(t test.Failer, path string) CallResponse
+	Call(path, host string) (CallResponse, error)
```

Comment:
Any chance we can align this with the Echo component's API?

Reply:
Are you talking about this API? https://github.com/istio/istio/blob/master/pkg/test/framework/components/echo/common/call.go
I can make this change in a separate PR.